clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -472,12 +475,17 @@ def _get_nodes_by_ip(self) -> Mapping[str, KubernetesNode]:
     def _get_pods_info(
         self,
     ) -> Tuple[
-        Mapping[str, List[KubernetesPod]], List[KubernetesPod], Mapping[str, List[KubernetesPod]], ClustermanResources
+        Mapping[str, List[KubernetesPod]],
+        List[KubernetesPod],
+        Mapping[str, List[KubernetesPod]],
+        ClustermanResources,
+        ClustermanResources,
```

Comment:
It might be the case to turn this into a namedtuple, as it's starting to get a bit difficult to read with all these returned things.